curl-config.in

```diff
@@ -174,7 +174,7 @@ while test $# -gt 0; do
 
     --static-libs)
         if test "X@ENABLE_STATIC@" != "Xno" ; then
-          echo @libdir@/libcurl.@libext@ @LDFLAGS@ @LIBCURL_LIBS@
+          echo "@libdir@/libcurl.@libext@" @LDFLAGS@ @LIBCURL_LIBS@
```

Comment:
Echo doesn't output these quotes anyway, does it?

Reply:
Done